src/OrbitGl/CaptureViewElement.h

```diff
@@ -76,6 +76,8 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   [[nodiscard]] virtual uint32_t GetLayoutFlags() const { return kScaleHorizontallyWithParent; }
   [[nodiscard]] virtual float DetermineZOffset() const { return 0.f; }
 
+  [[nodiscard]] bool LayoutHasChanged() const { return layout_has_changed_; }
```

Comment:
I think we would usually call this function `HasLayoutChanged`

Reply:
also the variable